crates/rome_js_semantic/src/semantic_model.rs

```diff
@@ -56,6 +56,18 @@ struct SemanticModelData {
 }
 
 impl SemanticModelData {
+    fn scope(&self, range: &TextRange) -> usize {
+        let scopes = self
+            .scope_by_range
+            .find(range.start().into(), range.end().into());
+
+        match scopes.last() {
+            Some(interval) => interval.val,
+            // We always have at least one scope, the global one.
+            None => unreachable!(),
```

Comment:
I would move this comment inside the unreachable: 

```suggestion
            None => unreachable!("We always have at least one scope, the global one."),
```

Reply:
Done.